app/src/main/java/com/example/activeamigo/CreateAccountActivity.java

```diff
@@ -0,0 +1,136 @@
+package com.example.activeamigo;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.EditText;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+import com.google.firebase.firestore.SetOptions;
+
+import java.util.HashMap;
+
+public class CreateAccountActivity extends AppCompatActivity {
+
+    protected FirebaseFirestore db;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_create_account);
+        String fSName = getResources().getString(R.string.dbAccounts);
+        db = FirebaseFirestore.getInstance();
+
```

Comment:
Adding Back Button: 

>>>ActionBar actionBar = getSupportActionBar();
>>>actionBar.setDisplayHomeAsUpEnabled(true); // showing the back button in action bar

Reply:
refer to https://github.com/CSE-210/Exercise-Buddy-App/blob/PreferencePage/app/src/main/java/com/example/activeamigo/PreferenceActivity.java#L68-L70